game/world/grassland_biome.lua

```diff
@@ -1,9 +1,10 @@
 local tallGrass = game.data.getVoxel("openbuilder_common_tallgrass")
 
 function onTopBlockSet(chunk, x, y, z, rng)
-    if rng < 20 then
-        createTree(chunk, x, y, z)
-    elseif rng < 50 then
+    local n = rng:nextInt(0, 3000)
+    if n < 20 then
+        createTree(chunk, x, y, z, rng)
+    elseif n < n then
```

Comment:
Grass doesn't spawn anymore due to this ;(

Reply:
Good catch :s fixed now